addons/hr_holidays/models/hr_leave_accrual_plan_level.py

```diff
@@ -134,6 +135,12 @@ class AccrualPlanLevel(models.Model):
         ('added_value_greater_than_zero', 'CHECK(added_value > 0)', 'You must give a rate greater than 0 in accrual plan levels.')
     ]
 
+    @api.constrains('maximum_leave')
+    def _check_maximum_leave(self):
+        for level in self:
+            if level.cap_accrued_time and not level.maximum_leave:
```

Comment:
Even though I don't think people would do it, I'd prevent negative numbers as well :eyes:
```suggestion
            if level.cap_accrued_time and level.maximum_leave < 1:
```